multiqc/utils/search_patterns.yaml

```diff
@@ -66,3 +66,5 @@ skewer:
     contents: 'COMMAND LINE:	skewer'
 samblaster:
     contents: 'samblaster: Version'
+samtools:
+    contents: 'samtools stats'
```

Comment:
Can we try to make this a bit more specific? I have a feeling that lots of other files might end up having `samtools stats` in.. Maybe something like `# This file was produced by samtools stats` instead?